patroni/ctl.py

```diff
@@ -149,9 +150,12 @@ def print_output(columns, rows=None, alignment=None, fmt='pretty', header=True,
         click.echo(t)
         return
 
-    if fmt == 'json':
+    if fmt in ['json', 'yaml']:
         elements = [dict(zip(columns, r)) for r in rows]
-        click.echo(json.dumps(elements))
+        if fmt == 'json':
+            click.echo(json.dumps(elements))
+        elif fmt == 'yaml':
+            click.echo(yaml.safe_dump(elements, encoding=None, allow_unicode=True, width=200))
```

Comment:
Should we use `default_flow_style=False` for showing the members list?  With a flow style the output is:

```
- {Cluster: batman, Host: 127.0.0.1, Lag in MB: 0, Member: postgresql0, Role: Leader, State: running}
- {Cluster: batman, Host: 127.0.0.1, Lag in MB: 0, Member: postgresql1, Role: '', State: running}

```
not much different from JSON